src/Config/Configuration.php

```diff
@@ -54,6 +54,11 @@ class Configuration
      */
     private $pathsToExclude = [];
 
+    /**
+     * @var list<string>
+     */
+    private $regexpsToExclude = [];
```

Comment:
Currently used naming is "regex", not "regexp". So I'd keep it consistent.

Reply:
https://github.com/shipmonk-rnd/composer-dependency-analyser/pull/140/commits/9953ffa0a931c9d9af4c208b5f72e6dd6ed24553